modules/destroy-hc-ibmpower-cli.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-destroy/hcp-destroy-ibmpower.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="destroy-hc-ibmpower-cli_{context}"]
+= Destroying a hosted cluster on {ibm-power-title} by using the CLI
+
+You can use the `hcp` command-line interface (CLI) to destroy a hosted cluster on {ibm-power-title}.
+
+.Procedure
+
+* Delete the hosted cluster and its backend resources by running the following command:
++
+[source,terminal]
+----
+$ hcp destroy cluster agent
+ --name=<hosted_cluster_name> \// <1>
+ --namespace=<hosted_cluster_namespace> <2>
```

Comment:
--cluster-grace-period is likely necessary. If I understand correctly (IIUC), the default value of 10m0s in the agent might not be sufficient to complete the deletion process, so we usually need to extend the time a bit.

Reply:
Updated the command to include the grace period flag and the description.